maisim/maisim.Game/Screen/Gameplay/Playfield.cs

```diff
@@ -42,39 +52,73 @@ public void SpawnTapNote(NoteLane lane)
         {
             AddInternal(new DrawableTapNote
             {
-                Anchor = Anchor.Centre,
-                Origin = Anchor.Centre,
                 Position = new Vector2(
-                    -(SPAWNER_MULTIPLIER * (float)Math.Cos((NoteLaneExtension.GetAngle(lane) + 90f) * (float)(Math.PI / 180))),
-                    -(SPAWNER_MULTIPLIER * (float)Math.Sin((NoteLaneExtension.GetAngle(lane) + 90f) * (float)(Math.PI / 180)))
-                    ),
-                Scale = new Vector2(1.2f),
-                Margin = new MarginPadding(10),
+                    -(SPAWNER_MULTIPLIER *
+                      (float)Math.Cos((NoteLaneExtension.GetAngle(lane) + 90f) * (float)(Math.PI / 180))),
+                    -(SPAWNER_MULTIPLIER *
+                      (float)Math.Sin((NoteLaneExtension.GetAngle(lane) + 90f) * (float)(Math.PI / 180)))
+                ),
                 Lane = lane
             });
         }
 
         protected override void Update()
         {
+            if (NotesPool != null)
+            {
+                foreach (Drawable note in NotesPool.ToList())
+                {
+                    // We have a lot of note type so we need to check on its type before we can update it
+                    if (note is DrawableTapNote tapNote)
+                    {
+                        if (tapNote.TargetTime != 0f && Clock.TimeInfo.Current >= tapNote.TargetTime - TIME_NOTE_APPEARS)
```

Comment:
Not sure about making a huge condition block for updating note position, sounds like something for an overridable method.

Reply:
Will try to move it as a method some like stratrgy.